### DISCUSSION TITLE
Permission to use images

### DISCUSSION THREAD

**Discussion by zoyahammad** [huggingface/diffusers#7989] [Q&A]
Hi, I'm a high school student researching on different text-to-image models. I would like to use one of your images from the following blog: https://huggingface.co/blog/stable_diffusion

However, my potential publisher (JEI) requires an official permission from the creators/owners beforehand. Could you please let me know who's the best person to contact for this? I was not able to find a relevant email address.

![stable_diffusion](https://github.com/huggingface/diffusers/assets/134557773/b1e4db5d-0b41-4784-955c-185b474fbfdb)

**Comment by yiyixuxu** [comment]
cc @pcuenca here!
